sdk/metric/exporter.go

```diff
@@ -34,12 +34,16 @@ type Exporter interface {
 	// This method needs to be concurrent safe with itself and all the other
 	// Exporter methods.
 	Temporality(InstrumentKind) metricdata.Temporality
+	// DO NOT CHANGE: any modification will not be backwards compatible and
+	// must never be done outside of a new major release.
```

Comment:
General comment

What is the value of these comments? Is it not obvious that (by default) an interface cannot change once the package is stable? Moreover we have https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#interface-stability

Reply:
This interface is not in the API and is not required to have methods added to it in the future. We therefore plan to address any change here in a backwards compatible Go manner.

Reply:
>  We therefore plan to address any change here in a backwards compatible Go manner.

That is why I would suggesting only adding comments to interfaces which are NOT handled in a backwards compatible Go manner. For me this comment only adds confusion 😉

Reply:
I'm not sure how adding a comment that says to not change things makes it unclear that that interface should not be changed. Can you explain?

Reply:
After the Go module is stable none method must be changed.

This comment makes me think: "Should this interface be handled differently?"

Reply:
This was a strategy the community decided on in https://github.com/open-telemetry/opentelemetry-go/issues/1714. This is ensuring consistency across signals and following the already instituted policy.